### ISSUE TITLE
cannot get my n64 configuration

### ISSUE THREAD

**Issue by FrostKnight** [mdqinc/SDL_GameControllerDB#807]
I tried to give you information recently, but perhaps I should have made my own thread like this.

The gamepad option you suggested, doesn't seem to take into account or detect L, R or Z buttons.

Or should I be going to the one maintaining that tool? I am not sure if its you or not.

**Comment by offalynne** [comment]
> *"The gamepad option you suggested, doesn't seem to take into account or detect L, R or Z buttons."*

I wrote you very specific instructions in the following comment.
https://github.com/mdqinc/SDL_GameControllerDB/pull/723#issuecomment-2484081523

**Comment by FrostKnight** [comment]
> > _"The gamepad option you suggested, doesn't seem to take into account or detect L, R or Z buttons."_
> 
> I wrote you very specific instructions in the following comment [#723 (comment)](https://github.com/mdqinc/SDL_GameControllerDB/pull/723#issuecomment-2484081523)

Yeah you did, but when I pressed L, R and Z I don't see anything happen. Every other button, shows something light up green. Its kind of puzzling.

**Comment by FrostKnight** [comment]
I think I figured out the problem on my end. Your setup expects the control stick in a different place, but I am connecting the star wars tie x wing ship shaped controller to the raphnet dual n64 v3 adapter.

The control stick is in an entirely different place. My apologies for not noticing that sooner.
